generators/example/src/main/scala/ConfigMixins.scala

```diff
@@ -33,23 +34,26 @@ import ConfigValName._
 // Common Parameter Mixins
 // -----------------------
 
-/**
- * Class to specify where the BootRom file is (from `rebar` top)
- */
 class WithBootROM extends Config((site, here, up) => {
   case BootROMParams => BootROMParams(
     contentFileName = s"./bootrom/bootrom.rv${site(XLen)}.img")
 })
 
-// DOC include start: WithGPIO
-/**
- * Class to add in GPIO
- */
+// DOC include start: gpio mixin
 class WithGPIO extends Config((site, here, up) => {
   case PeripheryGPIOKey => Seq(
     GPIOParams(address = 0x10012000, width = 4, includeIOF = false))
+  case BuildTop => (clock: Clock, reset: Bool, p: Parameters, success: Bool) => {
+    val top = up(BuildTop, site)(clock, reset, p, success)
+    for (gpio <- top.gpio) {
+      for (pin <- gpio.pins) {
+        pin.i.ival := false.B
```

Comment:
This seems weird. I would not expect this restriction based on the name of the mixin.

Reply:
While I agree, eventually this will be replaced by the IO cell blackbox work John mentioned... so maybe just have a comment saying this needs to be done or else FIRRTL will error?

Reply:
I guess so. It would be annoying if someone was currently tracking dev chipyard to do a real tapeout but I don't know of anyone doing that so it can wait.